pysensors/optimizers/_gqr.py

```diff
@@ -37,14 +40,25 @@ def __init__(self,idx_constrained,n_sensors,const_sensors,all_sensors):
             Column Indices of the sensors in the constrained locations.
         n_sensors : integer,
             Total number of sensors
-        const_sensors : integer,
+        n_const_sensors : integer,
             Total number of sensors required by the user in the constrained region.
+        all_sensors : np.ndarray, shape [n_features]
+            Optimall placed list of sensors obtained from QR pivoting algorithm.
+        constraint_option : string,
+            max_n_const_sensors : The number of sensors in the constrained region should be less than or equal to n_const_sensors.
+            exact_n_const_sensors : The number of sensors in the constrained region should be exactly equal to n_const_sensors.
```

Comment:
@niharika2999, the docstring needs to be updated. no `pivots` and `nx`, `ny`, `r` are not defined.